File: MORM.Java/src/MORM/Java/Models/Venda/Usuario.java
package MORM.Java.Models.Venda;

import java.util.Date;
import MORM.Java.Classes.*;
import MORM.Java.Classes.Mapping.*;

@TabelaAttribute(Nome = "USUARIO")
public class Usuario extends CollectionItem
{
    @CampoAttribute(Campo = "ID_USUARIO", Tipo = CampoTipo.Key)
    public Integer Id_Usuario;
    @CampoAttribute(Campo = "U_VERSION", Tipo = CampoTipo.Nul)
    public String U_Version;
    @CampoAttribute(Campo = "CD_OPERADOR", Tipo = CampoTipo.Req)
    public Integer Cd_Operador;
    @CampoAttribute(Campo = "DT_CADASTRO", Tipo = CampoTipo.Req)
    public Date Dt_Cadastro;
    @CampoAttribute(Campo = "NM_USUARIO", Tipo = CampoTipo.Req)
    public String Nm_Usuario;
    @CampoAttribute(Campo = "NM_LOGIN", Tipo = CampoTipo.Req)
    public String Nm_Login;
    @CampoAttribute(Campo = "CD_SENHA", Tipo = CampoTipo.Req)
    public String Cd_Senha;
    @CampoAttribute(Campo = "CD_PAPEL", Tipo = CampoTipo.Nul)
    public String Cd_Papel;
    @CampoAttribute(Campo = "TP_BLOQUEIO", Tipo = CampoTipo.Req)
    public Integer Tp_Bloqueio;
    @CampoAttribute(Campo = "DT_BLOQUEIO", Tipo = CampoTipo.Nul)
    public Date Dt_Bloqueio;
}

File: MORM.CSharp/Models/Venda/Usuario.cs
using MORM.CSharp.Classes;
using System;

namespace MORM.CSharp.Models
{
    [Tabela("USUARIO")]
    public class Usuario : CollectionItem
    {
        [Campo("ID_USUARIO", CampoTipo.Key)]
        public int Id_Usuario { get; set; }
        [Campo("U_VERSION", CampoTipo.Nul)]
        public string U_Version { get; set; }
        [Campo("CD_OPERADOR", CampoTipo.Req)]
        public int Cd_Operador { get; set; }
        [Campo("DT_CADASTRO", CampoTipo.Req)]
        public DateTime Dt_Cadastro { get; set; }
        [Campo("NM_USUARIO", CampoTipo.Req)]
        public string Nm_Usuario { get; set; }
        [Campo("NM_LOGIN", CampoTipo.Req)]
        public string Nm_Login { get; set; }
        [Campo("CD_SENHA", CampoTipo.Req)]
        public string Cd_Senha { get; set; }
        [Campo("CD_PAPEL", CampoTipo.Nul)]
        public string Cd_Papel { get; set; }
        [Campo("TP_BLOQUEIO", CampoTipo.Req)]
        public int Tp_Bloqueio { get; set; }
        [Campo("DT_BLOQUEIO", CampoTipo.Nul)]
        public DateTime Dt_Bloqueio { get; set; }
    }
}